README.md

```diff
@@ -1,48 +1,17 @@
-# Argos
+<p align="center">
+  <a href="https://argos-ci.com/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://raw.githubusercontent.com/argos-ci/argos/main/resources/logos/logo-github-readme.png" alt="Argos" width="300" height="61">
+  </a>
+</p>
 
-> Automate visual regression testing.
+_Argos is a visual testing solution that fits in your workflow to avoid visual regression. Takes screenshots on each commit and be notified if something changes._
 
-[![Build Status](https://travis-ci.org/argos-ci/argos.svg?branch=master)](https://travis-ci.org/argos-ci/argos)
+[![CI](https://github.com/argos-ci/argos/actions/workflows/ci.yml/badge.svg)](https://github.com/argos-ci/argos/actions/workflows/ci.yml)
 
-**Forget about regressions**:
-Argos will warn you if any visual regressions are about to be introduced,
-so they those don't end-up in production.
-We are giving developers high confidence in doing changes so they can quickly iterate.
-You can review visual changes in one click as part of your code review process.
+## Links
 
-**Save time**:
-Argos compares screenshots at high speed.
-You get a fast feedback.
-It comes with a GitHub integration.
-It will notify you on pull requests when something might be broken.
-
-**Integrates in your development workflow**:
-Argos integrates directly into your test suite and development workflow.
-We provide a command line interface streamlining the process.
-
-**Ship pixel-perfect interfaces**:
-Argos provides different tools to compare screenshots.
-Designers can easily participate in the code review process.
-
-**Testing your open source project is 100% free**:
-Seriously. Always. We like to think of it as our way of giving
-back to a community that gives us so much as well.
-
-## The problem solved
-
-- UI tests are **hard to write**.
-- Automated testing can't tell you if something doesn't look right. UI **regressions** may go undetected.
-- It's hard for **designers** to participate in the code review process.
-
-## Previous work
-
-- [happo](https://github.com/Galooshi/happo)
-- [spectre](https://github.com/wearefriday/spectre)
-- [wraith](https://github.com/BBC-News/wraith)
-- [BackstopJS](https://github.com/garris/BackstopJS)
-- [VisualReview](https://github.com/xebia/VisualReview)
-- [VisualReview-protractor](https://github.com/xebia/VisualReview-protractor)
-- [Shoov](https://github.com/shoov/shoov)
+- [Docs](https://docs.argos-ci.com)
```

Comment:
I propose to add a link to exemple folder here.

Reply:
Yeah I wanted but I forgot! Let's add it!

Reply:
Done!